VisitorTests/CodeGenerator/BuiltInFunctionsTests.cs

```diff
@@ -0,0 +1,51 @@
+﻿using GOAT_Compiler;
+using GOATCode.node;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using VisitorTests.Utilities;
+using Xunit;
+
+namespace VisitorTests
+{
+    public class BuiltInFunctionsTests
+    {
+        [SkippableTheory(typeof(TestDependencyException))]
+        [ClassData(typeof(GeneratesExpectedCodeEnumerator))]
+        public void CreatedFileChecker(string file)
+        {
+            // get the file and split it into the two sections
+            string TestFilePath = FileReadingTestUtilities.ProjectBaseDirectory + "CodeGenerator/Tests/InputFolder/" + file;
+            string fileContent = File.ReadAllText(TestFilePath);
+            string[] split = fileContent.Split('@');
+            string GOATCode = split.First();
+            string GCode = split.Last();
+
+            // do the parsing and checking needed
+            Start s = FileReadingTestUtilities.ParseString(GOATCode);
+            ISymbolTable symbolTable = FileReadingTestUtilities.BuildSymbolTable(s);
+            TypeChecker typeChecker = FileReadingTestUtilities.DoTypeChecking(s, symbolTable);
+            
+            // run the code generator
+            StringBuilder stringBuilder = new StringBuilder();
+            using (StringWriter writer = new StringWriter(stringBuilder))
+            {
+                CodeGenerator codeGenerator;
```

Comment:
Plz move the declaration down to the next line, this line is redundant :)

Reply:
Aka: remove this line and add "CodeGenerator" as prefix for the next line

Reply:
Fixed